modules/standard/Types.chpl

```diff
@@ -139,23 +169,38 @@ proc chpl__maxIntTypeSameSign(type t) type {
 // due to lack of consensus for the name.
 //
 
+pragma "no doc"
 proc isBoolValue(e)      param  return isBoolType(e.type);
+pragma "no doc"
 proc isIntValue(e)       param  return isIntType(e.type);
+pragma "no doc"
 proc isUintValue(e)      param  return isUintType(e.type);
+pragma "no doc"
 proc isRealValue(e)      param  return isRealType(e.type);
+pragma "no doc"
 proc isImagValue(e)      param  return isImagType(e.type);
+pragma "no doc"
 proc isComplexValue(e)   param  return isComplexType(e.type);
+pragma "no doc"
 proc isStringValue(e)    param  return isStringType(e.type);
+pragma "no doc"
 proc isIntegralValue(e)  param  return isIntegralType(e.type);
+pragma "no doc"
 proc isFloatValue(e)     param  return isFloatType(e.type);
+pragma "no doc"
 proc isNumericValue(e)   param  return isNumericType(e.type);
+pragma "no doc"
 proc isPrimitiveValue(e) param  return isPrimitiveType(e.type);
+pragma "no doc"
 proc isEnumValue(e)      param  return isEnumType(e.type);
 //Defined elsewhere:
 // isTupleValue
```

Comment:
Providing a cross-reference here might be useful to the maintainer. (Regular comment, not chpldoc comment.)